src/js/util-view/setVisibility.js

```diff
@@ -0,0 +1,7 @@
+export const setVisibility = ($target, isToBeVisible) => {
+  if (isToBeVisible) {
+    $target.hidden = false;
+  } else {
+    $target.hidden = true;
+  }
+};
```

Comment:
```suggestion
export const setVisibility = ($target, visibility) => {
  $target.hidden = !visibility;
};
```

또는

```suggestion
export const setHidden = ($target, hidden) => {
  $target.hidden = hidden
};
```

이렇게 짧게 작성하는게  더 직관적이겠죠?

Reply:
헛 한 줄로 더 가독성 좋게 작성할 수 있었군요..!! 감사합니다!! 👍👍👍